Microsoft/WebTemplateStudio
# fixed same name for cosmos and functions bug

This PR fixes #476 and enables the user to set same name for both azure functions and cosmos. 

_**Testing Notes:**_

- Launch the extension in VSCode
- Navigate to the services screen (select any folder location and project name)
- Add a cosmos deployment with any valid account name you like
- Add a functions deployment with the same app name you used for Cosmos account name. If the name isn't available for functions, change both cosmos account name and functions app name to a name available for both
- Navigate to summary page and hit `generate template`

Both Cosmos and functions should deploy now
